### PR TITLE
update fuzzit CLI to v2.4.57

### PR DESCRIPTION
**Pull request by yevgenypats**
This is a fix for https://github.com/envoyproxy/envoy/pull/7509#issuecomment-534384231

@lizan @htuch

### CODE REVIEW

**Comment by alyssawilk** [comment]
(This will need a master merge once #8345 lands to fix the format error.)

**Comment by yevgenypats** [comment]
done

File: ci/run_fuzzit.sh [hunk 1]
```diff
@@ -17,7 +17,7 @@ done
 
 
 # run fuzzing regression or upload to Fuzzit for long running fuzzing job ($1 is either local-regression or fuzzing)
-wget -O fuzzit https://github.com/fuzzitdev/fuzzit/releases/download/v2.4.55/fuzzit_Linux_x86_64
+wget -O fuzzit https://github.com/fuzzitdev/fuzzit/releases/download/v2.4.57/fuzzit_Linux_x86_64
```
Comment (lizan) [inline ci/run_fuzzit.sh]: @yevgenypats shall we always use latest release?

Comment (yevgenypats) [reply ci/run_fuzzit.sh]: looks like it working! well I'm not sure about it yet, lets use the pinned version for now.

**Pull request merged**